webapp/src/components/setting.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
Can we include the copyright header here? I see it's missing in a few other files as well. Curious how the linter didn't catch this.

Reply:
added everywhere

Reply:
Thanks for cleaning this up!